static/main.css

```diff
@@ -177,3 +177,18 @@ ul li button {
   position: static;
   margin-right: 8px;
 }
+
+.InputTransferKey {
+  display: grid;
+  grid-template-columns: 100px 10px 100px 10px 100px 10px;
+  grid-gap: 10px;
+  padding-bottom: 10px;
+}
+
+.InputTransferKey .seperator {
+  text-align: center;
```

Comment:
is this `text-align: center` needed? I wonder if we could create then a class `.centered` or use `styled-components` or something..